BetSystem/BusinnessLogic/SportEventService.cs

```diff
@@ -62,5 +67,98 @@ public void DeleteById(int id)
             _dbContext.Events.Remove(sportEvent);
             _dbContext.SaveChanges();
         }
+
+        public void AddTeamToSportEvent(int id, IdRequestDto idRequestDto)
+        {
+            var team = _dbContext.Teams.FirstOrDefault(x => x.Id == idRequestDto.Id);
+            _dbContext
+                .Events.FirstOrDefault(x => x.Id == id)
+                .Teams.Add(team);
+            _dbContext.SaveChanges();
+        }
+
+        public List<TeamDto> GetTeamsFromSportEvent(int id)
+        {
+            return _dbContext.Events
+                .Include(x => x.Teams)
+                .FirstOrDefault(x => x.Id == id)
+                .Teams
+                .Select(x => new TeamDto
+                {
+                    Id = x.Id,
+                    Name = x.Name,
+                }).ToList();
+        }
+
+        public void AddBetToSportEvent(BetOnEventDto betOnEventDto)
+        {
+            var betEventToSave = new BetOnEvent
+            {
+                BetOnResult = betOnEventDto.BetOnResult,
+                Amount = betOnEventDto.Amount,
+                Interest = betOnEventDto.Interest,
+                AmountToPay = betOnEventDto.AmountToPay,
+                Team = _dbContext.Teams.First(x => x.Id == betOnEventDto.TeamId),
+                Event = _dbContext.Events.First(x => x.Id == betOnEventDto.EventId)
+            };
+            _dbContext.Bets.Add(betEventToSave);
+            _dbContext.SaveChanges();
+        }
+
+        public List<BetOnEventDto> GetBetsFromSportEvent(int id)
+        {
+            return _dbContext.Events
+                .Include(x => x.Bets)
```

Comment:
Can we go from Bets.Where(x => x.Events.Id == id)? Or we don't have such relation?

Reply:
Don't have such relation.